hack/testing/check-zzz-fluentd-prometheus-scrape.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+
+LOGGING_NS=${LOGGING_NS:-openshift-logging}
+
+fpod="$(get_running_pod fluentd)"
+
+scrape_output="$(oc exec ${fpod} -- curl http://localhost:24231/metrics)"
```

Comment:
`curl -s` to avoid the download stats output

Reply:
Also - is it possible to hit the endpoint from outside the pod e.g. can you do `curl -s http://${what_goes_here}:24231` ?